Pipfile

```diff
@@ -31,10 +30,7 @@ publish = "twine upload dist/*"
 format = "bash -c \"python -m isort setup.py pygate_grpc tests && python -m black $(git ls-files '*.py')\""
 lint = "bash -c \"python -m flake8\""
 integration-test = "python -m pytest tests/integration/"
-test-ffs = "python -m pytest tests/integration/test_ffs.py"
+test-api = "python -m pytest tests/integration/test_api.py"
 
 [packages]
-grpcio = "*"
-protobuf = "*"
-six = "*"
-deprecated = "*"
+grpc-powergate-client = "==1.1.2"
```

Comment:
This brings along all those other dependencies

Reply:
Thanks for the PR @asutula we really appreciate it!!! I will take a look as well, work on the integration tests and merge asap!

Reply:
The PR looks great to me!! Thanks for your contribution! However, I believe there is some stuff still missing before releasing v1.0.0 to be compatible with the new API. I will merge this PR in a new branch and add what's missing in there so we can release the new version! :)